CONTRIBUTING.md

```diff
@@ -102,7 +102,10 @@ We also offer other commands that offer you assistance during your local develop
   - **Usage:** `npx turbo scripts:release-post -- --version=vXX.X.X --force`
 - `npx turbo storybook` starts Storybook's local server
 - `npx turbo storybook:build` builds Storybook as a static web application for publishing
-- `npx turbo test` runs jest (unit-tests) locally
+- `npx turbo test` runs all tests locally
```

Comment:
I think you should also on the unit testing section of the guidelines mention that unit testing of components rendering as expected is done via Storybook. And add maybe a link to some reference/docs

Reply:
Sounds good! I'll update those tomorrow. Almost 5am here.

Reply:
Okay I'll be waiting for that :)

Reply:
Working on it! Got sick over the weekend.

Reply:
Updated!